### DISCUSSION TITLE
Run multiple commands under one comment (headline)

### DISCUSSION THREAD

**Discussion by ridicolos** [dbeaver/dbeaver#14787] [Q&A]
So I was wondering if there is a way to run multiple commands under one headline.

For example: I want to have one script which does multiple things for testing

```sql
-- CREATE ENVIRONMENT
DROP SCHEMA IF EXISTS test CASCADE;
CREATE SCHEMA test;
CREATE TABLE test.names(name varchar);

-- Insert some test values
insert into test.names values ('mario');
insert into test.names values ('luigi');
```

I don't want to select all the code under the comment and run it manually. I would like to press on the headline-comment and just run all below this headline. This works for one command (directly under the comment, but not the ones below that). 
From the docs I couldn't figure out on how to do that. 

Is there a way to start a block and end it?

**Comment by ShadelessFox** [comment]
Hello @ridicolos,

I'm not sure about the profit from this feature.

Executing everything below just a single comment is a potentially hazardous operation.

**Reply by ridicolos** [reply]
Yea. After a little big of thinking I came to the same conclusion. A comment should only describe one command.

But for some other usecases: Is there any other way to define something in "blocks"?

**Reply by ShadelessFox** [reply]
It's not possible from a DBeaver perspective, but you can use PL/SQL blocks, for example. In such case, DBeaver can also fold them.

**Reply by ridicolos** [reply]
Alright thanks! I will try that out.
